build/sir-dev/Dockerfile

```diff
@@ -0,0 +1,61 @@
+ARG PYTHON_VERSION=2.7
+FROM metabrainz/python:${PYTHON_VERSION}
+
+ARG PYTHON_VERSION
+
+LABEL org.metabrainz.based-on-image="metabrainz/python:${PYTHON_VERSION}"
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+#######################
+# From metabrainz/sir #
+#######################
+
+RUN apt-get update \
+    && apt-get install --no-install-recommends -qy \
+      build-essential \
+      ca-certificates \
+      cron \
+      git \
+      # TODO: check if this is actually needed
+      libffi-dev \
+      # required for connections of pip packages
+      libssl-dev \
+      # required for psycopg2. Installs without it but links against a wrong .so.
+      libpq-dev \
+      # TODO: check if this is actually needed
+      libsqlite3-dev \
+      # TODO: check if this is actually needed
```

Comment:
libxml2-dev is required by libxslt1-dev. https://packages.debian.org/buster/libxslt1-dev

Reply:
* Removed `libxml2-dev` in https://github.com/metabrainz/musicbrainz-docker/pull/173/commits/4240a7e3a35ac01f18fc4ad98666854e3145fb5f (actual reference is https://packages.ubuntu.com/xenial/libxslt1-dev)
* Kept `libsqlite3-dev` in https://github.com/metabrainz/musicbrainz-docker/pull/173/commits/47a167cf8713b0a0286d449671bcd1f2ff50bccc (as it is required for test)